Library/Homebrew/cmd/update-report.rb

```diff
@@ -209,8 +209,13 @@ def report
         end
         @report[:M] << tap.formula_file_to_name(src)
       when /^R\d{0,3}/
-        @report[:D] << tap.formula_file_to_name(src) if tap.formula_file?(src)
-        @report[:A] << tap.formula_file_to_name(dst) if tap.formula_file?(dst)
+        next unless tap.formula_file?(dst)
```

Comment:
I just realized this check can be dropped completely, as there's a check preceding the `case` statement that already makes sure that at least one of the modified paths is a formula file:

```ruby
next unless paths.any? { |p| tap.formula_file?(p) }
```